### PR TITLE
Fix single coop issues

### PR DESCRIPTION
**Pull request by wvengen**
- non-background mailing didn't work
- the error handler had an issue

#788

### CODE REVIEW

File: app/controllers/errors_controller.rb [hunk 1]
```diff
@@ -11,15 +11,6 @@ def show
...
 
-  def select_foodcoop
-    foodcoop = params[:foodcoop]
-    if FoodsoftConfig.allowed_foodcoop? foodcoop
-      FoodsoftConfig.select_foodcoop foodcoop
-    else
-      FoodsoftConfig.select_default_foodcoop
-    end
-  end
-
```
Comment (wvengen) [inline app/controllers/errors_controller.rb:22]: This is already provided by the `ApplicationController`, and that handles single coop installs better.

Comment (paroga) [reply app/controllers/errors_controller.rb:22]: i'd create a separate commit for that, since it's unrelated

File: lib/foodsoft_config.rb [hunk 2]
```diff
@@ -75,6 +75,11 @@ def init(filename = APP_CONFIG_FILE)
       self.scope = config[:default_scope] or raise "No default_scope is set"
       # Set defaults for backward-compatibility
       set_missing
+      # When multicoops is enabled, the foodcoop is selected on each request.
+      # When multicoops is disabled, we need to do some initialization.
+      unless config[:multi_coop_install]
```
Comment (paroga) [inline lib/foodsoft_config.rb:80]: do we really need the `unless`. if not, i'd omit it

**Review by paroga — APPROVED** [review]

File: lib/foodsoft_config.rb (hunk 2 above)
Comment (wvengen) [reply lib/foodsoft_config.rb:80]: We don't (I'll test) :+1:

File: app/controllers/errors_controller.rb (hunk 1 above)
Comment (wvengen) [reply app/controllers/errors_controller.rb:22]: Ah, thanks - I intended to but then forgot!

**Pull request merged**